### ISSUE TITLE
Refinements to Energy Graph AccordionBox

### ISSUE THREAD

**Issue by arouinfar** [phetsims/energy-skate-park#262]
The Energy vs. Position/Time AccordionBox currently looks like this:
![image](https://user-images.githubusercontent.com/8419308/80066009-5f121d80-84f8-11ea-861b-1bc1009e6fe0.png)

I have some concerns with the current legend/energy type checkboxes.
1. The color contrast between the text and gray background is poor. Should we use black text and a color swatch instead?
2. There isn't a lot of real estate for i18n, and we've already abbreviated "KE" and "PE". Should we add an info dialog, as was done in Masses and Springs? The dialog would allow translators to decide if/how to abbreviate each of the energy types.

Here's a rough mockup:
![image](https://user-images.githubusercontent.com/8419308/80066465-376f8500-84f9-11ea-9e6e-05661d04ab45.png)

**Comment by arouinfar** [comment]
4/23/20 design meeting notes:

* Use "Kinetic" and "Potential" like we do in the bar graph and pie chart legend. This would eliminate the need for an Energy dialog.
* Expand AccordionBox so it is left-aligned with the Grid/RefHeight panel so string have more room for i18n.
* Vertically align color dots.
* Reduce zoom button size by 10% (but maintain the same touch areas).
![image](https://user-images.githubusercontent.com/8419308/80147746-367a3a00-8571-11ea-8915-002e1e75189a.png)

**Comment by arouinfar** [comment]
Energy Graph AccordionBox pixel polishes (moved from #240).

- [ ] Increase size of AccordionBox by expanding it to the left (grid position should not change) so it is left-aligned with Grid/Reference Height panel below.
- [ ] Increase the left-padding of ExpandCollapseButton to match its top-padding. (Shift it to the right 1-2 px)
- [ ] Left-align energy type checkboxes with ExpandCollapseButton.
- [ ] Change "KE" to "Kinetic" and "PE" to "Potential".
- [ ] Make energy type checkbox labels size 12 pt font and black.
- [ ] Add a colored circle next to each energy type string corresponding to the color of the data in the graph. Vertically align these icons with one another (see mockup in comment above).
- [ ] Reduce size of energy type checkboxes to match the smaller size used elsewhere in the sim.
- [ ] Vertically center the Erase button between the top boundary of the AccordionBox and the graph. It's currently very close to the top border of the AccordionBox.
- [ ] Increase size of font used in Position/Time ABSwitch to match the size used for the graph axis labels. (Looks like it could be 14 pt, maybe?)
- [ ] Reduce size of zoom buttons by 10%, but leave the touch areas the same size.

**Comment by jessegreenberg** [comment]
@arouinfar another thing related to the accordion box...Do you think the title bar should be clickable? This sim puts controls in the title bar and so if you click near a control but not quite on it, the box will collapse. I have found this to be a little annoying sometimes.

**Comment by jessegreenberg** [comment]
Here is an example, clicking to the left of the ABSwitch toggle, but to the right of the label closes the box.
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/6396244/80402402-dd214c00-888b-11ea-8556-99d171c2bfd5.gif)

**Comment by arouinfar** [comment]
I've also had similar frustrations with accidentally closing the AccordionBox. Does AccordionBox have an option to make the title bar unpickable? I know there has been some back and forth on this in https://github.com/phetsims/sun/issues/477 and https://github.com/phetsims/sun/issues/502. Given the presence of other UI elements in the title bar, I think it's appropriate for the title bar to be unpickable in this sim.

For ABSwitch, you should be able to click on the node on either side to toggle the switch. Looks like that is working, but my cursor remains the default arrow when hovering over the text. @jessegreenberg can you look into this as well?

**Comment by jessegreenberg** [comment]
> Does AccordionBox have an option to make the title bar unpickable

It does, done in the above commit.

> Looks like that is working, but my cursor remains the default arrow when hovering over the text.

You are right! I am seeing this for all ABSwitches in the project, taking to sun.

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/energy-skate-park/issues/262#issuecomment-620161307 should both be fixed.

**Comment by jessegreenberg** [comment]
> Increase size of AccordionBox by expanding it to the left (grid position should not change) so it is left-aligned with Grid/Reference Height panel below.

This was done naturally with the addition of the energy circles. But it will be tricky to align with the grid/reference height panel because that panel floats with wider screens. At default dev bounds the two are now reasonably aligned.

@arouinfar that completes the list in https://github.com/phetsims/energy-skate-park/issues/262#issuecomment-618667971, can you please review how the graph accordion box looks in master now?

**Comment by arouinfar** [comment]
@jessegreenberg it's looking really good in master! Just one last polish request -- can you reduce the size of the erase button by 10%? It's looking just a bit too crowded and could use some more padding. The touch area size can remain as it is now.

![image](https://user-images.githubusercontent.com/8419308/80434033-e7584000-88b5-11ea-8f8c-80978c03f1d8.png)

**Comment by jessegreenberg** [comment]
Sure, it was reduced 10%. How is that?

**Comment by arouinfar** [comment]
Looks great, thanks!
